[
    {
        "content": "<p>I have a CI error I don't understand at <a href=\"https://github.com/leanprover-community/mathlib/pull/10713\">#10713</a> . At the<code>export as low-level text file</code> step of the <code>Run tests</code> CI, I get the error </p>\n<div class=\"codehilite\" data-code-language=\"Lean\"><pre><span></span><code><span class=\"n\">Run</span> <span class=\"n\">lean</span> <span class=\"c1\">--recursive --export=mathlib.txt src/</span>\n<span class=\"bp\">&lt;</span><span class=\"n\">unknown</span><span class=\"bp\">&gt;</span><span class=\"o\">:</span><span class=\"mi\">1</span><span class=\"o\">:</span><span class=\"mi\">1</span><span class=\"o\">:</span> <span class=\"n\">error</span><span class=\"o\">:</span> <span class=\"n\">unknown</span> <span class=\"n\">declaration</span> <span class=\"bp\">'</span><span class=\"n\">_private.3627408217.add'</span>\n<span class=\"n\">Error</span><span class=\"o\">:</span> <span class=\"n\">Process</span> <span class=\"n\">completed</span> <span class=\"k\">with</span> <span class=\"n\">exit</span> <span class=\"n\">code</span> <span class=\"mi\">1</span><span class=\"bp\">.</span>\n</code></pre></div>\n<p>(You can go to the PR and click on <code>details</code> on the failing CI for more info)<br>\nI ran the tests several times because I thought it might go away on its own, but the error stays. Does anyone know what I should do about it?</p>",
        "id": 264824844,
        "sender_full_name": "Rémy Degenne",
        "timestamp": 1639470482
    },
    {
        "content": "<p>Might be a cache bug - merging master might be enough to invalidate whatever bad cache it's using</p>",
        "id": 264826445,
        "sender_full_name": "Eric Wieser",
        "timestamp": 1639471524
    },
    {
        "content": "<p>Thanks I'll try that</p>",
        "id": 264826643,
        "sender_full_name": "Rémy Degenne",
        "timestamp": 1639471682
    }
]